packages/manager/modules/bm-server-components/src/consumption-tile/controller.js

```diff
@@ -0,0 +1,15 @@
+export default class QuotaTileController {
+  /* @ngInject */
```

Comment:
```suggestion
  
```
should be remove because there is no injection in the controller

Reply:
I applied.